src/Illuminate/Foundation/Console/OptimizeCommand.php

```diff
@@ -38,6 +39,11 @@ class OptimizeCommand extends Command
      */
     public function handle()
     {
+        $confirmationMessage = 'Further configuration and routes changes will not take effect unless you run the command again or clear the cache, are you sure you want to run this command?';
+        if (App::environment('local') && ! $this->confirm($confirmationMessage)) {
```

Comment:
`App::isLocal()` can be used instead of `App::environment('local')`.

https://github.com/laravel/framework/commit/4e963fdd1144e940f05bb2357fdbf4c8dbb2a584#diff-0a80606049a107748776bea5151a61fbec2afefe3361c340a5de9c8c1ccdf9c8

Reply:
Made this change, thanks!